tests/test_ec2.py

```diff
@@ -548,6 +548,33 @@ def test_ec2_rename_tag(self):
         resources = policy.run()
         self.assertEqual(len(resources), 3)
 
+    def test_ec2_multiple_tags(self):
+        localtz = zoneinfo.gettz('America/New_York')
+        dt = datetime.now(localtz).replace(year=2018, month=3, day=15)
+        session_factory = self.replay_flight_data('test_ec2_multiple_tags')
+        policy = self.load_policy({
+            'name': 'multiple-tags-ec2',
+            'resource': 'ec2',
```

Comment:
could you use a different resource on the record, ec2 are a bit verbose on their describe outputs.